src/ngraph/runtime/cpu/op/batch_mat_mul_transpose.hpp

```diff
@@ -34,14 +34,17 @@ namespace ngraph
         class BatchMatMulTranspose : public Op
         {
         public:
+            NGRAPH_API
+            static const std::string type_name;
```

Comment:
Any reason this is puablic if we have `description()` ?

Reply:
With it public you can write something like `BatchMatMulTrasnpose::type_name` instead of typing the error prone literal string `"BatchMatMulTrasnpose"`. It is just a safer way to do it.